### PR TITLE
Add splash:get_version()

### PR DESCRIPTION
**Pull request by Youwotma**

### CODE REVIEW

File: docs/scripting-ref.rst [hunk 1]
```diff
@@ -1822,6 +1822,30 @@ Example 3 - extract all cookies set by website without reading response body
...
     end
 
+.. _splash-version:
+
+splash:get_version
+------------------
+
+Get Splash mayor and minor version.
+
+**Signature:** ``splash:get_version()``
```
Comment (kmike) [inline docs/scripting-ref.rst]: major, minor = splash:get_version()

**Comment by kmike** [comment]
I think 'major' is common spelling. Otherwise, +1 to merge.

**Comment by Youwotma** [comment]
Right, commit amended!

**Comment by kmike** [comment]
Hey @Youwotma,

What do you think about returning a Lua table instead of 2 values? We can extend table with more information: major/minor/point, "dev" suffix, qt version, python version.

**Pull request merged**